### DISCUSSION TITLE
Font size problem

### DISCUSSION THREAD

**Discussion by JonPolish** [freeplane/freeplane#1528] [General]
I must be missing something. Whenever staring new child nodes, I get a large font in the first created node. Siblings are all smaller. 
Of course I can correct it, but the problem repeats. 

Thank you.

Jon

![2023-11-14](https://github.com/freeplane/freeplane/assets/121563385/23851f12-fa43-46a9-a1e7-cd9792ea5386)

**Comment by dpolivaev** [comment]
Nobody can help you unless you zip and attach your mind map

**Comment by JonPolish** [comment]
Oops. Sorry.

Jon

[Demo.zip](https://github.com/freeplane/freeplane/files/13352407/Demo.zip)

___________________________________________________________

On Tuesday, November 14, 2023, 11:18:21 AM, Dimitry Polivaev wrote:





Nobody can help you unless you zip and attach your mind map

**Reply by EdoFro** [reply]
try unchecking the last two options from the Format menu:

![image](https://github.com/freeplane/freeplane/assets/35700575/c5b8f6df-e61a-4702-9cd8-38b1d3d4ad59)

**Reply by JonPolish** [reply]
Thank you so much. So simple and effective. I appreciate your help.

Jon
